core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/sink/managed/ProgressiveSinkOpDesc.java

```diff
@@ -56,7 +56,7 @@ public OperatorInfo operatorInfo() {
                 "View the edu.uci.ics.texera.workflow results",
                 OperatorGroupConstants.RESULT_GROUP(),
                 asScalaBuffer(singletonList(new InputPort("", false))).toList(),
-                List.empty(), false, false, false);
+                List.empty(), false, false, false, false);
```

Comment:
as we have many boolean instances as positional arguments, it's better to annotate them of the argument name. In java, if keyword argument is not allowed, maybe try to use comments to annotate them?